profiling/tests/phpt/gc_mem_caches_01.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+[profiling] test allocation profiling not blocking `gc_mem_caches()` call
+--DESCRIPTION--
+Directly after rinit, the `gc_mem_caches()` function is able to cleanup some
+60KB of data, but with a custom allocator installed in the ZendMM it won't do
+anything and return `int(0)`. As we do prepare the heap for this call, the
+function should actually cleanup some memory and return the amount in bytes.
+--SKIPIF--
+<?php
+if (!extension_loaded('datadog-profiling'))
+    echo "skip: test requires Datadog Continuous Profiler\n";
+?>
+--ENV--
+DD_PROFILING_ENABLED=yes
+DD_PROFILING_EXPERIMENTAL_CPU_TIME_ENABLED=yes
+DD_PROFILING_EXPERIMENTAL_ALLOCATION_ENABLED=yes
```

Comment:
Probably add `DD_PROFILING_LOG_LEVEL=off` to this?

Reply:
See #1876